package.json

```diff
@@ -2,12 +2,20 @@
   "name": "ten-project-test",
   "version": "0.1.0",
   "private": true,
+  "lint": "eslint .",
+  "lint:fix": "eslint --fix",
```

Comment:
if this is supposed to be a script, we should have this include `--ext` like in your doc